File: wrappers/dotnet/indy-sdk-dotnet/PoolApi/PoolLedgerConfigExistsException.cs
namespace Hyperledger.Indy.PoolApi
{
    /// <summary>
    /// Exception thrown when attempting to create a pool ledger config with same name as an existing pool ledger config.
    /// </summary>
    public class PoolLedgerConfigExistsException : IndyException
    {
        const string message = "A pool ledger configuration already exists with the specified name.";

        /// <summary>
        /// Initializes a new PoolLedgerConfigExistsException.
        /// </summary>
        internal PoolLedgerConfigExistsException() : base(message, (int)ErrorCode.PoolLedgerConfigAlreadyExistsError)
        {

        }
    }

}

File: wrappers/android/src/main/java/org/hyperledger/indy/sdk/pool/PoolLedgerConfigExistsException.java
package org.hyperledger.indy.sdk.pool;

import org.hyperledger.indy.sdk.ErrorCode;
import org.hyperledger.indy.sdk.IndyException;

/**
 * Exception thrown when attempting to create a pool configuration with the same name as one that already exists.
 */
public class PoolLedgerConfigExistsException extends IndyException
{
	private static final long serialVersionUID = 2032790158242533689L;
	private final static String message = "A pool ledger configuration already exists with the specified name.";

	/**
	 * Initializes a new PoolLedgerConfigExistsException.
	 */
	public PoolLedgerConfigExistsException()
	{
		super(message, ErrorCode.PoolLedgerConfigAlreadyExistsError.value());
	}
}